db/changes.go

```diff
@@ -400,7 +400,7 @@ func (db *Database) changesFeed(singleChannelCache SingleChannelCache, options C
 				paginationOptions.Limit = base.MinInt(remainingLimit, queryLimit)
 			}
 
-			// TODO: pass db.Ctx down to changeCache?
+			// TODO: pass db.LoggingCtx down to changeCache?
```

Comment:
unintentional change

```suggestion
			// TODO: pass db.Ctx down to changeCache?
```